tests/test_jobs.py

```diff
@@ -2665,6 +2665,22 @@ def test_scheduler_do_not_schedule_too_early(self):
 
         queue.put.assert_not_called()
 
+    def test_scheduler_do_not_schedule_without_tasks(self):
+        dag = DAG(
+            dag_id='test_scheduler_do_not_schedule_without_tasks',
+            start_date=DEFAULT_DATE)
+
+        with create_session() as session:
+            orm_dag = DagModel(dag_id=dag.dag_id)
+            session.merge(orm_dag)
+            session.commit()
```

Comment:
`.commit()` and `.close()` can be omitted since they are part of the `create_session`

Reply:
Thanks for catching that, removed suggested lines.